### DISCUSSION TITLE
ND issues route- just for info

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6888] [General]
Hi developers,

just for info, the drawing on the ND with a programmed VOR DME app on RWY 32 with teardrop .... 

![grafik](https://user-images.githubusercontent.com/76531893/157857748-a99d3065-778d-4ec1-86ff-08c4016bf5c9.png)

EDIT: Drawing is better in the near of the VOR:

![grafik](https://user-images.githubusercontent.com/76531893/157859891-21ce6263-cf03-470e-af12-a8497d74ce2a.png)

greets

**Comment by tracernz** [comment]
That will resolve when you slow down... for the moment it uses live TAS for turn prediction, which causes these overshoots (which are realisticly depicted for Honeywell... but once predictions are available the real one uses them to draw it properly).
